dockers/docker-database/docker_init.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Generate supervisord config file
+mkdir -p /etc/supervisor/conf.d/
+
+script="/usr/bin/create_all_redis_config.sh"
+conf_file="/etc/sonic/database_config.json"
```

Comment:
>"/etc/sonic/database_config.json" [](start = 10, length = 33)

I believe this is static for each image, and we are not allowed to change it after deployed. If this is a good assumption/requirement, we may render templates during building image.

Reply:
If I understand correctly, you mean the database_config file is static , that means this config file is a part of the image and we already decided how to allocate the databases into each instance during building image time? After the deployment, there should NOT have a way to change the config file until we release next version(if any database instances changes are needed).  If this is true , then for other comments we can ignore the template at run time and we can generate all those scripts at compiling/building time. Is that correct?